src/registries/auto_completes/pivot_auto_complete.ts

```diff
@@ -70,7 +70,7 @@ autoCompleteProviders.add("pivot_measures", {
 
     return definition.measures
       .map((measure) => {
-        if (measure.name === "__count") {
+        if (measure.fieldName === "__count") {
```

Comment:
the measure autocomplete should auto-complete the id, but it autocompletes the field name